mlir/test/Dialect/Arith/int-range-interface.mlir

```diff
@@ -731,18 +731,23 @@ func.func @extui_uses_unsigned(%arg0 : i32) -> i1 {
     func.return %4 : i1
 }
 
-/// Catch a bug that crash in getLoopBoundFromFold when
-/// SparseConstantPropagation loaded in the solver.
-
-// CHECK-LABEL: func.func @caller(%[[ARG0:.*]]
-// CHECK: call @callee(%[[ARG0]])
-// CHECK-LABEL: func.func @callee(%[[ARG0]]
-// CHECK: %[[LOAD:.*]] = affine.load %[[ARG0]]
-// CHECK: scf.for {{.*}} to %[[LOAD]] 
```

Comment:
This is just cleaning up parts of the test that were removed earlier, right?

Reply:
Yep, adapting to what the automatic tool generates. Previous was wrong as you cannot have variables in labels in FileCheck